docs/integrations/scheduled-commands.md

```diff
@@ -0,0 +1,60 @@
+---
+id: scheduled-commands
+title: Scheduled Commands
+---
+
+### Available from XSOAR version 6.2.0 and above.
+
+<img width="533" src="../doc_imgs/integrations/polling-command.png"></img>
+
+It's possible for a command to schedule a future execution for another command with a predetermined `next_run` time, `args` as well as a `timeout`.
+
+Use cases for using scheduled commands include:
+1. ***Polling Flow*** - The command cannot return the full result in a single execution (likely because it's waiting for a remote process to finish execution). Scheduled commands enable to set the command to try again later, and return the full result when it can. Example use cases are `Sandbox Detonation` and `Autofocus samples search`.
+2. ***Rate Limiting*** - The command cannot perform the required action because the instance has reached its rate limit. Enable Scheduled commands to try again later in order to check if the rate limit has been removed. If so, the command can be executed.
+3. ***Concurrency*** - The command cannot perform the required action because the instance has reached its concurrent limit. Scheduled commands are enabled to set the command to try again later (preferably with exponential backoff).
+
+### YAML Prerequisite
+* ***Integration***: In the integration yml, under the command root add `polling: true`.
+* ***Script***: In the script yml, in the root of the file add `polling: true`.
+
+For an example, see the Autofocus V2 `autofocus-samples-search` command.
+
+### ScheduleMetadata
+`ScheduleMetadata` is an optional class that enables scheduling commands via the command results.
+
+#### ScheduleMetadata Class Arguments
+| Arg               | Type   | Description                                                                                                                                                                                |
+|-------------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| command                | str    | The command that will run after `next_run_in_seconds` has passed.
+| next_run_in_seconds    | int    | How long to wait before executing the command.
+| args                   | dict   | Arguments to use when executing the command.
+| timeout_in_seconds     | int    | Number of seconds until the polling sequence will timeout.
+
+When provided to [CommandResults](./code-conventions#commandresults) it will transform its result into a ***schedule result***.
+When the time comes for the next command to run, it will be executed.
+The scheduled command can return another ***schedule result***, that will schedule another schedule command and so on.
+
+The schedule sequence will be complete when either one of three terminating actions happen:
+
```

Comment:
add a note that the minimum interval time is 10 seconds.

Reply:
mention also that the playbook will proceed only when the polling command/script will finish.
And that the commands will use workers only when executed. Between executions the workers released.